### ISSUE TITLE
Environment variable in command->argument

### ISSUE THREAD

**Issue by lprat** [DFIR-ORC/dfir-orc#30]
Hi,

ORC don't allow to insert environment variable in command->argument.
E.G:
```
<command keyword="sigcheck.exe system32">
            <execute name="sigcheck.exe" run32="7z:#Tools|sigcheck.exe" run64="7z:#Tools|sigcheck64.exe"/>
            <argument>-accepteula -h -e -a -c %windir%\system32</argument>
            <output name="system32_infos.csv" source="StdOut"/>
            <output name="system32_infos.log" source="StdErr"/>
</command>
```
Please, can you improve this feature, by allow var env in argument or by directory in "input".

Thanks.
Lionel

**Comment by fabienfl-orc** [comment]
Hi, should be available from 10.0.23. Thank you

**Comment by lprat** [comment]
Sorry ! It's fixed! Thanks
